testcases/MachineConfig.py

```diff
@@ -44,7 +45,7 @@ def setUp(self):
 
         conf = OpTestConfiguration.conf
         self.cv_SYSTEM = conf.system()
-        self.cv_SYSTEM.goto_state(OpSystemState.OFF)
```

Comment:
just wanted to understand the why line removed ?

Reply:
As my class OsConfig, for configuring hugepage(2M,16M and 1G) doesn't require lpar shutdown.. So as the above line is a part of main setup so I have moved this to individual Classes where the other Classes need lpar shutdown in order to perform their operations